zebra/zebra_router.h

```diff
@@ -46,6 +47,7 @@ struct zebra_router_table {
 
 	struct route_table *table;
 };
+
```

Comment:
Remove this

Reply:
updated in https://github.com/FRRouting/frr/pull/9440/commits/a5819304e72c8fee7521fab8aa64852b75871c7b